exercise/static/exercise/results_staff.js

```diff
@@ -349,7 +349,7 @@
                     }
                 }
             }
-            newHeading += (rowStart + _(summaries[item]['TITLE']) + '</td><th></th><th>' + summArray[item][TOTAL_COL_ID] + '</th>' + cells + rowEnd);
+            newHeading += (rowStart + _(summaries[item]['TITLE']) + '</td><th></th><th></th><th>' + summArray[item][TOTAL_COL_ID] + '</th>' + cells + rowEnd);
```

Comment:
It seems like one more extra cell should be added (as this change added both an extra name column as well as the organization column). The summary cells seem to be off by one:
![image](https://user-images.githubusercontent.com/50318434/128013710-74b2a375-4233-4e30-bb68-2842a611b140.png)

Reply:
Oh, never mind. This problem is caused because the header cells or the summary rows span only 2 columns and not 3 (as they should).